src/main/java/org/example/LowerCase.java

```diff
@@ -0,0 +1,8 @@
+package org.example;
+
+class LowerCase implements TextFormatter {
```

Comment:
LowerCaseFormatter - myślę, że taka nazwa lepiej by pasowała, to samo dotyczy tej drugiej klasy